pages/index.js

```diff
@@ -2,12 +2,14 @@ import {
   FsaLayout,
   SessionWrapper,
   ContentItem,
-  ContinueButton
+  ContinueButton,
+  BrowserUnsupportedBanner
 } from "../src/components";
 import { Header, Paragraph, InsetText } from "govuk-react";
 
 const Index = props => (
   <FsaLayout {...props}>
+    <BrowserUnsupportedBanner supportedBrowser={props.supportedBrowser} />
```

Comment:
Instead of passing argument to the component, just check if it's supported here, then render it or nothing based on that check.